config/custom_items.txt

```diff
@@ -155,6 +155,11 @@ character_name: Dameon Owen
 item_path: /obj/item/weapon/reagent_containers/food/snacks/cookie/mysterious
 }
 {
+ckey: dameonowen
```

Comment:
Don't need to update legacy, plus this does not actually remove the old entry just making a duplicate of it

Reply:
nvm i see you adding it to a new char

Reply:
nvm seeing you adding it to a new char